tgui/packages/tgui/styles/interfaces/CloningConsole.scss

```diff
@@ -0,0 +1,55 @@
+@use '../colors.scss';
+@use '../components/Button.scss';
+
+.Cloning-Console_FlexTable {
```

Comment:
From a "make use of SCSS" standpoint, you can write this whole file within the "block" of `.cloning-console`, so it would look something like this:
```
.cloning-console {
  &__flex-table {
    display: flex;
    // ...
  }

  &__flex-head {
    display: flex;
    // ...
  }

  &__head-row {
    display: flex;
    // ...
  }

  // ...
}